Supporting Documents/BS_SD.adoc

```diff
@@ -1229,147 +1198,213 @@ In this scenario, at least 30,000 imposter transactions shall be conducted with
+. Sample acceptance error tradeoff
++
+The TOE needs to make decisions about whether or not to accept a sample for further processing based on quality metrics. Such decisions are subject to Type I/II error tradeoff analysis from decision theory. First, Type I error expresses incorrect rejection of a good biometric sample, i.e. assignment of low-quality when the sample would be enrolled or verified by the TOE correctly; and Type II error expresses incorrect acceptance of a low-quality sample when it ultimately gives a false negative.
 +
-The determination of the quality score that will be used as the basis for sufficient quality.
+Both error rates of TOE’s quality metrics should be low enough to serve as an indicator of the usefulness of the biometric sample for biometric enrolment and verification.
++
+The detail information of both Type I and II errors is described in, for example, <<Ongoing Face Recognition Vendor Test>>.
+
+The developer shall conduct the test following either of above approaches to show the efficacy of TOE’s quality metrics and create the test report. 
+
+The report is most likely a separate confidential document and not part of the ST for public release.
+
+==== Summary of contents
+<<nonstdQualityItemsTable>> shows items that shall be reported in the report. The name or structure of report does not need to follow <<nonstdQualityItemsTable>>. However, all items in <<nonstdQualityItemsTable>> should be included somewhere in the report. If some items are not included in the report, the developer shall provide a rationale for such exclusion to the evaluator.
+
+:xrefstyle: short
+[cols="1,2",options="header",]
+.Quality Assessment Criteria Report Items (non-standard quality metrics)
+[[nonstdQualityItemsTable]]
+|===
+|*Section* 
+|*Item*
+
+|<<Test configuration>> 
+|Test configuration
+
+|<<Quality metrics and expected test result>> 
+|Quality metrics and expected test result
+
+|<<Test result of TOE quality metrics>> 
+|Test result of TOE quality metrics
+
+|<<Methods to create low-quality samples>> 
+|Methods to create low-quality samples
 
-==== Quality algorithm identification
-The developer shall report on the algorithm(s) used to assess the quality of biometric samples. 
+|===
+
+:xrefstyle: full
 
-The discussion of the algorithms(s) should include information about the defects that are considered and handled in determining quality. The specific algorithm(s) used to generate a quality score, whether general or modality-specific, must be covered. 
+==== Quality assessment criteria report (non-standard quality metrics)
+This Section describes each item in <<nonstdQualityItemsTable>> in detail. Most of items are created based on <<Reporting items description>> so the developer may refer them to create the report.
 
-The following information shall be included in the report.
+==== Test configuration
 
+Same as <<Test configuration>>
```

Comment:
You can't reference sections with the same name (in this case headings) as it isn't clear which one it will be linked to (in the post processing it could go to either heading). This needs to be done with anchor markers, but I'm not clear which line this needs to anchor to readily.

Reply:
10.2.3. Test configuration
Same as Section 10.1.3, “Test configuration”

Yes I understand that but content of "10.2.3. Test configuration" should the same one as described in "10.1.3. Test configuration" so we don't change 1363.

Reply:
OK, I'll mark it as resolved for now and look at it again after everything is merged.